src/bindings/python/src/pyopenvino/frontend/inputmodel.cpp

```diff
@@ -361,4 +362,25 @@ void regclass_frontend_InputModel(py::module m) {
                 type : ngraph.Type
                     New element type.
             )");
+
+    im.def(
+        "set_tensor_value",
+        [](ov::frontend::InputModel& self, ov::frontend::Place::Ptr& place, py::array& value) {
```

Comment:
if possible:
```suggestion
        [](ov::frontend::InputModel& self, const ov::frontend::Place::Ptr& place, const py::array& value) {
```

Reply:
Partially fixed, `py::array& value` cannot be fixed as it is converted to C-style array.